.github/workflows/examples-nrfconnect.yaml

```diff
@@ -53,17 +53,23 @@ jobs:
               run: scripts/examples/nrfconnect_example.sh shell nrf52840dk_nrf52840
             - name: Build example nRF Connect SDK Pigweed on nRF52840 DK
               run: scripts/examples/nrfconnect_example.sh pigweed-app nrf52840dk_nrf52840
+            - name: Build example nRF Connect SDK Lock App on nRF5340 DK
```

Comment:
should we consider to build only one example per new platform have CI move somewhat faster?